docs(hub): add debug section (#4417)
jina-ai/jina: docs/fundamentals/executor/hub/debug-executor.md

# Debug Executor

You need to debug Jina Hub Executor for some reason. It's not so easy to debug inside the Docker container, but Jina Hub Executor provides a way to use source code. It's easy to debug using source code. 

> I'll take debugging [Hello Executor](https://hub.jina.ai/executor/9o9yjq1q) as an example. Tell you step by step how to do that. 

Reviewer: ```suggestion
I'll use [Hello Executor](https://hub.jina.ai/executor/9o9yjq1q) as an example for debugging and show you step by step how to do that. 
```
